test/client-options-test.js

```diff
@@ -14,6 +14,7 @@ describe('SOAP Client', function() {
     'overrideRootElement': {
       'namespace': 'tns'
     },
+    'overridePromiseSuffix': 'test',
```

Comment:
for readability within the test assertion:  would you mind to capitalize the suffix to `Test`? 👍